tests/cypress/support/commands.js

```diff
@@ -0,0 +1,55 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import '@10up/cypress-wp-utils';
+import { getRandomText } from "../support/functions";
+
+Cypress.Commands.add( 'visitAdminPage', ( page = 'index.php' ) => {
+	cy.login();
+	if ( page.includes( 'http' ) ) {
+		cy.visit( page );
+	} else {
+		cy.visit( `/wp-admin/${ page.replace( /^\/|\/$/g, '' ) }` );
+	}
+});
```

Comment:
We have a conversation [here](https://github.com/10up/cypress-wp-utils/pull/1) about not adding the `visitAdminPage` command. I think we should follow the conclusion there in our plugin as well. What do you think? Also cc @cadic.

Reply:
Done.